docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -16,18 +16,18 @@ More information about the main changes between the two resources can be found [
+Having that in mind, to access all the latest functionalities and stay up to date with our best offering we recommend you to start planning your move to `mongodbatlas_advanced_cluster`. To maintain our focus on enhancing the overall experience with `mongodbatlas_advanced_cluster`, `mongodbatlas_cluster` has npw been deprecated starting from provider version 2.0.0, and will soon be removed from the provider completely. Begin your planning now to ensure you're ready for this transition.
 
-### What is the `mongodbatlas_advanced_cluster` Preview of MongoDB Atlas Provider 2.0.0?
+### What was the `mongodbatlas_advanced_cluster` Preview of MongoDB Atlas Provider 2.0.0?
 
-To make it easier to migrate to `mongodbatlas_advanced_cluster`, we decided to enable support for the [`moved` block](https://developer.hashicorp.com/terraform/language/moved). This functionality needs the resource to be implemented using the [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework), whereas our existing implementation [uses the SDKv2](https://developer.hashicorp.com/terraform/plugin/sdkv2). Given the considerable changes between the two frameworks and the breaking changes it causes, we have decided to release a preview version of the `mongodbatlas_advanced_cluster` usable under an environment variable and keep the existing implementation as-is to avoid breaking existing users. 
-Once the MongoDB Atlas Provider 2.0.0 is released, only the new version will remain and the environment variable won't be needed.
-More information about the preview version of `mongodbatlas_advanced_cluster` can be found in the [resource documentation page](../resources/advanced_cluster%2520%2528preview%2520provider%25202.0.0%2529).
+To make it easier to migrate to `mongodbatlas_advanced_cluster`, we decided to enable support for the [`moved` block](https://developer.hashicorp.com/terraform/language/moved). This functionality needed the resource to be implemented using the [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework), whereas our previous implementation [used the SDKv2](https://developer.hashicorp.com/terraform/plugin/sdkv2). Given the considerable changes between the two frameworks and the breaking changes it causesd we decided to release a preview version of the `mongodbatlas_advanced_cluster`, in our provider versions 1.29.0 and later, usable under an environment variable and kept the existing implementation as-is to avoid breaking existing users. 
```

Comment:
```suggestion
To make it easier to migrate to `mongodbatlas_advanced_cluster`, we enabled support for the [`moved` block](https://developer.hashicorp.com/terraform/language/moved). This functionality requires you to implement the resource using the [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework). However, our previous implementation [used the SDKv2](https://developer.hashicorp.com/terraform/plugin/sdkv2). Given the considerable changes between the two frameworks and the breaking changes it causes, we releases a preview version of the `mongodbatlas_advanced_cluster`, in our provider versions 1.29.0 and later, which you can use under an environment variable and keep the existing implementation as-is to avoid breaking existing implementations. 
```